zcncore/get_data.go

```diff
@@ -169,6 +171,30 @@ func withParams(uri string, params Params) string { //nolint:unused
 //	}, nil)
 //}
 
+// GetConfig retrieves the configuration of the smart contract
+//   - configType: the type of configuration to retrieve (e.g. storage_sc_config, miner_sc_globals, miner_sc_configs)
+func GetConfig(configType string) ([]byte, error) {
```

Comment:
It should be in the transaction pakage. Why not use from there direclty? It will be further utilised by mobile_sdk.go only right?

Reply:
Sure I can move this to GetConfig.

Reply:
Done